### PR TITLE
find the media based on the tags provided

### PR DESCRIPTION
**Pull request by xxu511**
Finish the function that based on tags could find the media. The function would find all medias that contain the tags provided by user in specific watchlist. For example, in test1 watchlist, there are 2 medias-"HHH"with tags movie,show,"XXX" with tag movie. If user input the tag movie would get both media, if user input the move and show would return "HHH" media.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ async def help(interaction: nextcord.Interaction):
     response += "/choose \n- select a random item from a watchlist\n\n"
     response += "/status \n- update watch status of a movie or show in a watchlist\n\n"
     response += "/history \n- display all the watched movies in a watchlist\n\n"
+    response += "/filter_tags \n- display all the filtered movies based on tagsin a watchlist\n\n"
```
Comment (Mman755) [inline bot.py]: Nice work adding this in for your functionality, however just a very minor typo that needs to be addressed.

```suggestion
    response += "/filter_tags \n- display all the filtered movies based on tags in a watchlist\n\n"
```

File: bot.py [hunk 2]
```diff
@@ -538,4 +540,41 @@ async def watchlist_history(interaction: nextcord.Interaction, watchlist_name):
...
 
+
+"""
+Displays all the media that filtered based on tags in a watchlist.
+Args:
+    interaction: The interaction object representing the command invocation.
+    watchlist_name: The name of the watchlist to which the watched movies going to be displayed
+    tags: The tags that want to be based on to filter
+
+Returns:
```
Comment (Mman755) [inline bot.py:551]: Good work adding in this documentation, I think you might have missed specifying the return type, good job though.

```suggestion
Returns:
None
```

File: bot.py [hunk 3]
```diff
@@ -538,4 +540,41 @@ async def watchlist_history(interaction: nextcord.Interaction, watchlist_name):
...
+
+"""
+@bot.slash_command(guild_ids=[GUILD_ID], name="filter_tags", description="view filtered medias based on tags in a watchlist")
```
Comment (Mman755) [inline bot.py]: I think filter_by_tags might make more sense for the users unfamiliar with the bot :)

```suggestion
@bot.slash_command(guild_ids=[GUILD_ID], name="filter_by_tags", description="view filtered medias based on tags in a watchlist")
```

**Comment by Mman755** [comment]
Great job on the implementation, It works well. Just some minor changes on your behalf and it should be good to merge. Great work!

**Comment by xxu511** [comment]
the pull request would closes #61

**Review by Mman755 — APPROVED** [review]
Great work with the little changes and overall implementation.

**Pull request merged**